modules/stanford_basic_page_types/lib/scss/component/stanford-page/_basic-page-types.scss

```diff
@@ -0,0 +1,239 @@
+@charset "UTF-8";
+
+//
+// Custom Styles For Stanford Page Content Type.
+//
+
+.stanford-page {
+  &.su-page-description {
+    margin-bottom: 46px;
+
+    @include grid-media-min("l") {
+      max-width: 85ch;
+    }
+  }
+
+  @include grid-media-min("l") {
+    .field--name-su-list-view {
+      &.field--type-viewfield {
+        padding-left: 0;
+        padding-right: 0;
+      }
+    }
+  }
+}
+
+// basic list styles
+.su-basic-page-types-list {
```

Comment:
This should be pulled out into it's own stylesheet that is added to the view

Reply:
Including all the other styles related to the view display.

Reply:
This is fixed.